resources/nginx.conf

```diff
@@ -108,6 +108,7 @@ http {
 
         # API
         location /api {
+            proxy_set_header email-header smostovoy@nextdoor.com;
```

Comment:
Did you mean to leave this hardcoded to your email?

Reply:
Oops... good catch.